app/commands/get_recommendations.rb

```diff
@@ -0,0 +1,24 @@
+class GetRecommendations < PowerTypes::Command.new(:github_user, :organization)
+  def perform
+    membership = OrganizationMembership.where(github_user_id: @github_user.id,
+                                              organization_id: @organization.id).first
+    if membership
+      if membership.is_member_of_default_team
+        GetReviewRecommendations.for(
+          github_user_id: @github_user.id,
+          other_users_ids: other_default_team_members_id
+        )
+      end
+    end
+  end
+
+  private
+
+  def other_default_team_members_id
+    OrganizationMembership
+      .where(organization_id: @organization.id, is_member_of_default_team: true)
+      .pluck(:github_user_id)
+      .reject { |id| id == @github_user.id }
+      .sort
```

Comment:
esto puede quedar como `OrganizationMembership.where(organization_id: @organization.id, is_member_of_default_team: true).where.not(id: @github_user.id).sort`